tests/x11/gnomecase/nautilus_permission.pm

```diff
@@ -65,6 +66,7 @@ sub run {
     send_key "r";        #choose properties
     assert_screen 'nautilus-properties';
     send_key "up";       #move focus onto tab
+    send_key "up";       #in some cases it needs one more hit, to reach the tab
```

Comment:
Mind leaving a comment or a hint:

If problem keeps happening, switch to an assert and click strategy (on both lines)

Reply:
Hm, actually I could add a condition only for TW, as there it is needed two clicks up to reach the tab. Would it be better like this?